guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,375 @@
+      alt: .
+      caption: .
+    - file: send-address-error
+      modalImage: send-address-error@2x
+      alt: .
+      caption: .
+
+images_test-txns:
+    - file: test-txn-send-review
+      modalImage: test-txn-send-review@2x
+      alt: .
+      caption: .
+    - file: test-txn-send-home
+      modalImage: test-txn-send-home@2x
+      alt: .
+      caption: .
+    - file: test-txn-confirm-finalise-txn
+      modalImage: test-txn-confirm-finalise-txn@2x
+      alt: .
+      caption: .
+
+images_backup:
+    - file: backup-cloud-backup
+      modalImage: backup-cloud-backup@2x
+      alt: .
+      caption: .
+    - file: backup-wallet-birthday
+      modalImage: backup-wallet-birthday@2x
+      alt: .
+      caption: .
+
+images_recovery:
+    - file: recovery-wallet-birthday-prompt
+      modalImage: recovery-wallet-birthday-prompt@2x
+      alt: .
+      caption: .
+    - file: recovery-wallet-birthday-enter
+      modalImage: recovery-wallet-birthday-enter@2x
+      alt: .
+      caption: .
+---
+
+<!--
+
+Editor's notes
+
+Privacy is not a crime.
+
+Thanks to josie (josibake@protonmail.com) for his invaluable work and guidance.
+
+This page provides UX insights, best practices and designs for silent payments.
+
+Source designs for all screens can be found in this Figma file:
+https://www.figma.com/design/q8IYl9UVsOWkkPhs4IkHQR/Bitcoin-Wallet-UI-Kit-%26-Design-System-(SP-updates)?node-id=5621-52155&t=4frDIaY1YrPQurMl-1
+Source file for header image:
+https://www.figma.com/design/q8IYl9UVsOWkkPhs4IkHQR/Bitcoin-Wallet-UI-Kit-%26-Design-System-(SP-updates)?node-id=5873-1050&t=4frDIaY1YrPQurMl-1
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/silent-payments-header.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/silent-payments-header@2x.png"
+   mobile = "/assets/images/guide/how-it-works/silent-payments/silent-payments-header-mobile.png"
+   mobileRetina = "/assets/images/guide/daily-spending-wallet/silent-payments/silent-payments-header-mobile@2x.png"
+   alt-text = "A header image with a purple gradient, the text Silent Payments and a network visual."
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Silent payments
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+[Silent payments](https://github.com/bitcoin/bips/blob/master/bip-0352.mediawiki) is a protocol involving static addresses that are used to derive unique on-chain address during every transaction. This prevents [address reuse](/guide/glossary/address/#address-reuse) without  repeated user interaction, enabling an interaction model for on-chain transactions centred around users.
+
+For eg: Alice, who runs an NGO, can simply post a static address on her website, and receive bitcoin donations at unique on-chain addresses that only she can identify. The static address itself never shows up on-chain.
```

Comment:
I personally found this a bit confusing. 
`The static address itself never shows up on-chain.` 

I assume the static address in this case referring to the Silent Payment address? The one that begins with SP? It's clear after I think about it, but I had to stop and make sure that 'the static address' means the silent payment address.

Reply:
I've adopted the static address terminology since it explains how the address world and avoid questions like what is a silent payment or what's silent about the payment/address. Tried to ensure I used the same terminology everywhere on this page in https://github.com/BitcoinDesign/Guide/pull/1109/commits/985c8c4d94f3060759e7f50eb1bae0ed7d778356